### PR TITLE
Do not sign the .js files

### LINKED ISSUE #69288: [wasm] verify template javascript files don't include signatures
Install the public preview4 and verify the template files are not signed.

### PR DESCRIPTION
**Pull request by radical**
We want to avoid the js files in wasm templates from getting the
signature block in user visible files.

Fixes https://github.com/dotnet/runtime/issues/69288 .

### CODE REVIEW

**Review by akoeplinger — APPROVED** [review]

File: eng/Signing.props [hunk 1]
```diff
@@ -20,6 +20,8 @@
     <!-- apphost and comhost template files are not signed, by design. -->
     <FileSignInfo Include="apphost.exe;singlefilehost.exe;comhost.dll" CertificateName="None" />
 
+    <FileExtensionSignInfo Include=".js" CertificateName="None" />
```
[the remark this answers is not in the document]
Comment (radical) [reply eng/Signing.props:23]: Do we know that this applies to all the .js files here?
These are the non-wasm, and non-tests ones:

```
./eng/actions/backport/index.js
./src/native/libs/System.Native/pal_random.lib.js
```

**Review by mmitche — APPROVED** [review]

**Comment by radical** [comment]
The llvmfullaot failure is https://github.com/dotnet/runtime/issues/69330 .

**Comment by agocke** [comment]
@radical This looks like it has broken the official build

**Comment by radical** [comment]
[Build](https://dev.azure.com/dnceng/internal/_build/results?buildId=1771861&view=logs&j=413a2d4f-ed72-5d06-f334-66633ae63807&t=ce65dcc1-e5a7-5b8b-b4f4-df7bb0ee9f83):

```
D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\7.0.0-beta.22255.2\tools\Sign.proj(50,5):
error : Multiple certificates for extension '.js' defined for CollisionPriorityId ''.  There should be one certificate per extension per collision priority id.
```

Only the coreclr/windows builds seem to be broken. Maybe we should restrict the change to only when building for wasm.

**Comment by radical** [comment]
Which step does the signing for wasm nugets?

**Comment by hoyosjs** [comment]
That usually happens post build - in a separate repo.

**Comment by mmitche** [comment]
I think this is because *.js already has a default certificate from the Sign.props file. You want to replace it with "None", rather than adding a "None"

**Pull request merged**